.travis.yml

```diff
@@ -56,6 +56,12 @@ jobs:
         - make .gitvalidation
         - make verify
         - make binaries
+      go: 1.9.x
```

Comment:
Let's just remove 1.8.x
Both Kubernetes and containerd moved to 1.9 already.

Reply:
Done.